concourse/scripts/build_tarball.bash

```
@@ -261,9 +264,11 @@ build_upload_log_srv() {
     cd ${log_sv_src}
     mkdir -p LogService/bin
     mkdir build && cd build
-    local cmake_args="-DCMAKE_BUILD_TYPE=$BUILD_TYPE -DWITH_ASAN=$ASAN -DDISABLE_CODE_LINE_IN_LOG=ON -DUSE_ROCKSDB_LOG_STATE=ON"
+    local cmake_args="-DCMAKE_BUILD_TYPE=$BUILD_TYPE -DWITH_ASAN=$ASAN -DDISABLE_CODE_LINE_IN_LOG=ON"
     if [ "$ds_type" = "ELOQDSS_ROCKSDB_CLOUD_S3" ]; then
-        cmake_args="$cmake_args -DWITH_ROCKSDB_CLOUD=S3 -DWITH_CLOUD_AZ_INFO=ON"
+        cmake_args="$cmake_args -DWITH_LOG_STATE=ROCKSDB_CLOUD_S3 -DWITH_CLOUD_AZ_INFO=ON"
+    elif [ "$kv_type" = "ELOQDSS_ROCKSDB_CLOUD_GCS" ]; then
```

Shellcheck (0.11.0)

[warning] kv_type is referenced but not assigned.

(SC2154)